### ISSUE TITLE
Ability to choose extension for magnet files

### ISSUE THREAD

**Issue by Napsterbater** [Sonarr/Sonarr#2165]
Simply an option for TorrentBlackhole to save the .magent file as a .url

Either a 2nd toggle that can be triggered after "Save Magnet Files" is set to yes. Such as "Save as .url"

or

A 2nd toggle independent of the first, with the option "Save Magnet Files as .url" and maybe the orginal as "Save Magnet Files as .magnet"

Or if its not to much harder. the 2nd/new option (which ever way is decided) could be left open to what ever is needed, where the user specifiys a file extension to be used.

Bottom line, I need a .url file vs a .magnet file as tixati only accepts .url and .desktop, just thinking of ways that could happen. I am approaching this

**Comment by markus101** [comment]
This feels like a can of worms. Since every client could require something different for the extension. If this is something we end up doing either a dropdown or a free form input would be best I think.

You can work around this using a custom script that runs `On Grab` and updates any `.magnet` file in the Blackhole folder and changes it to `.url` or `.desktop` (why the hell is that a thing?).
